xbmc/cores/VideoPlayer/DVDSubtitles/DVDSubtitlesLibass.h

```diff
@@ -36,14 +53,87 @@ class CDVDSubtitlesLibass
   */
   int GetNrOfEvents() const;
 
+  /*!
+  * \brief Decode Header of ASS/SSA, needed to properly decode
+  * demux packets with DecodeDemuxPkt
+  * \return True if success, false if error
+  */
   bool DecodeHeader(char* data, int size);
+
+  /*!
+  * \brief Decode ASS/SSA demux packet (depends from DecodeHeader)
+  * \return True if success, false if error
+  */
   bool DecodeDemuxPkt(const char* data, int size, double start, double duration);
+
+  /*!
+  * \brief Create a new empty ASS track
+  * \return True if success, false if error
+  */
+  bool CreateTrack();
```

Comment:
Isn't this used only by the adapter? If so move it near the other private methods. If not please ignore